rero_ils/modules/acq_invoices/jsonschemas/acq_invoices/acq_invoice-v0.0.1.json

```diff
@@ -0,0 +1,307 @@
+        "acq_account",
+        "document",
+        "order_number",
+        "quantity",
+        "price"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "acq_order_line": {
+          "title": "Acquisition order line",
+          "type": "object",
+          "properties": {
+            "$ref": {
+              "title": "Acquisition order line URI",
+              "type": "string",
+              "pattern": "^https://ils.rero.ch/api/acq_order_lines/.*?$"
+            }
+          }
+        },
+        "acq_account": {
+          "title": "Acquisition account",
+          "type": "object",
+          "properties": {
+            "$ref": {
+              "title": "Acquisition account URI",
+              "type": "string",
+              "pattern": "^https://ils.rero.ch/api/acq_accounts/.*?$"
+            }
+          }
+        },
+        "document": {
+          "title": "Document",
+          "type": "object",
+          "properties": {
+            "$ref": {
+              "title": "document URI",
+              "type": "string",
+              "pattern": "^https://ils.rero.ch/api/documents/.*?$"
+            }
+          }
+        },
+        "order_number": {
+          "title": "Order number",
+          "type": "string",
+          "minLength": 3
+        },
+        "quantity": {
+          "title": "Numbor of units",
+          "type": "integer",
+          "minimum": 1
+        },
+        "price": {
+          "title": "Price per unit",
+          "type": "number",
+          "minimum": 0
+        },
+        "discount": {
+          "title": "Discount amount",
+          "type": "number",
+          "minimum": 0
+        },
+        "total_price": {
+          "title": "Total price",
+          "type": "number",
+          "minimum": 0
+        }
+      }
+    }
+  },
+  "properties": {
+    "$schema": {
+      "title": "Schema of the resource",
+      "description": "Schema to validate Acquisition Invoice records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/acq_invoices/acq_invoice-v0.0.1.json",
+      "pattern": "^https://ils.rero.ch/schema/acq_invoices/acq_invoice-v([0-9]+?\\.){3}json$"
+    },
+    "pid": {
+      "title": "Acquisition Invoice ID",
+      "type": "string",
+      "minLength": 1
+    },
+    "invoice_number": {
+      "title": "Invoice number",
+      "type": "string",
+      "minLength": 1,
+      "form": {
+        "validation": {
+          "validators": {
+            "valueAlreadyExists": {
+              "term": "invoice_number"
+            }
+          },
+          "messages": {
+            "alreadyTakenMessage": "The invoice number is already taken"
+          }
+        }
+      }
+    },
+    "invoice_price": {
+      "title": "Invoice price",
+      "type": "number"
+    },
+    "invoice_status": {
+      "title": "Invoice status",
+      "type": "string",
+      "enum": [
+        "in progress",
```

Comment:
usually we do not use space for code. Can you check the other JSONSchema for consistency? This should be documented somewhere.

Reply:
In the document schema (L831) space is used.